server/src/utils/call.ts

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import FormData from 'form-data';
+import { try_later } from './queue';
+import { is_retryable } from './retry';
+
+export async function endpointPost(endpoint_url: string, data: FormData, extra: any) {
+    return await httpCall({ 
+        method: 'post',
+        url: endpoint_url,
+        data: data,
+        headers: extra}, true);
+}
+
+export async function endpointGet(endpoint_url: string, req_qry: any, token: string) {

```

Comment:
I'm thinking something more like... `getFhirResource` might be a better naming schema here

Reply:
I think we only would want to do retries when requesting fhir data, I don't think we want to introduce them into the auth flow

Reply:
it's a unified code so we have a centralized place to add control, in case something else needed, it can be added here conveniently.